engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -410,43 +409,25 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
     @Command(shortDescription = "Your ping to the server", helpText = "The time it takes the packet " +
         "to reach the server and back", requiredPermission = PermissionManager.NO_PERMISSION)
     public String ping(@Sender EntityRef sender) {
+        Server server = networkSystem.getServer();
+        if (server == null) {
+            //TODO: i18n
```

Comment:
that's true 😆 I thought `String.format` looked better with the longer messages, but that might just be a personal thing...